### ISSUE TITLE
[mesheryctl] `registry generate` meshkit error for component generation failure

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#11040]
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

### Current Behavior
```
Current model: couchbase-operator
yaml: unmarshal errors:
  line 13374: mapping key "exposeWebhookPort" already defined at line 13372
Current model: cpx
  extracted 40 components for Contrail Analytics (contrail-analytics)
```

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- 📺 [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- ⌨️ [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- ⌨️ [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by Jougan-0** [comment]
> ```
> Current model: couchbase-operator
> yaml: unmarshal errors:
>   line 13374: mapping key "exposeWebhookPort" already defined at line 13372
> Current model: cpx
>   extracted 40 components for Contrail Analytics (contrail-analytics)
> ```
> 
> The error is in of their CRDs, (redefined same property)

Yes, but don't you think we need a better handling that this something like error exist at xyz line due to the helm chart on the repository has a syntax error rather than just showing this.

**Comment by leecalcote** [comment]
@Jougan-0 will you see that `registry` is fully promoted from `exp` and is no longer listed under this section in Meshery Docs? It is listed in two places currently.
